roles/python/molecule/default/verify.yml

```diff
@@ -0,0 +1,29 @@
+---
+# Tests for python role
+
+- name: verify
+  hosts: all
+  vars_files: ../../defaults/main.yml
+  gather_facts: false
+
+  tasks:
+    - name: check python version
+      command: python --version
+      register: _python_version
+      changed_when: _python_version
+
+    - name: python version should be default
+      assert:
+        that: _python_version.stdout.startswith("Python 3.6")
+
+    - name: check virtual environment
+      stat:
+        path: "{{ python_venv_path }}"
+      register: _venv
+
+    - name: virtual environment should exist
+      assert:
+        that: _venv.stat.isdir
+
+
+    
```

Comment:
should the verify check installing python requirements also? probably not too hard to create a requirements file with one or two things (but is it bad to install things just for testing?)

Reply:
totally agree, and as soon as i started testing this i found out that the tests don't do what I thought they did 😬 so...more tests coming